main.go

```diff
@@ -245,21 +251,23 @@ func main() {
 			os.Exit(1)
```

Comment:
The webhook for KubeadmControlPlane should likely be included in the block below as well.

Reply:
argh, thx. fixed in amended commit.